include/n_menu/N_Menu.cpp

```diff
@@ -258,13 +257,11 @@ GMEXPORT double N_Menu_CleanUp(){
     }
     numMenus = 0;
     menuId = 2000;
-    for(unsigned int i = 0; i < numBitmaps; i++){
-        if(bitmaps[i] != 0){
+    for(unsigned int i = 0; i < bitmaps.size(); i++){
             DeleteObject(bitmaps[i]);
-            bitmaps[i] = 0;
-        }
+	    bitmaps.erase(bitmaps.cbegin()+i);

```

Comment:
This won't work because erase() changed the size of the vector. Better would be to go through the vector and free all the bitmaps first, then clear the vector in one go (i.e. set its size to zero).
In fact, since this is C++, you could use a C++ style for loop to do it.